src/components/DialogBox/CombinationalAnalysis.vue

```
@@ -214,64 +214,114 @@ function createLogicTable() {
     }
 }
 
+function printBooleanTable() {
+    // Create a new div to hold the table content
+    const tableContent = document.createElement('div')
+    tableContent.innerHTML = tableHeader.value.map((header, index) => `
```

ESLint

[error] 'index' is defined but never used.

(no-unused-vars)